packages/vendure-plugin-shipping-extensions/README.md

```diff
@@ -0,0 +1,64 @@
+# Shipping Extenions Vendure Plugin
+
+### [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-shipping-extensions)
+
+This plugin does two things in general
+
+- adds a shipping eligibility checker to Vendure that checks the total weight and the shipping country of an
+  order, to verify if a shipping method is eligible for a given order.
+- introduces a distance based (`ShippingCalculator`)[https://docs.vendure.io/reference/typescript-api/shipping/shipping-calculator/], based on a configurable `OrderAddressToGeolocationConversionStrategy` used to convert the `shippingAddress` of an `Order` to geographic latitudinal and longitudinal values.
+
+The weight of a product can be configured on the customfield `Product.weight`. You can configure the units to be in KG,
+grams or whatever unit you like.
+
+A Custom `OrderAddressToGeolocationConversionStrategy` can be configured by as follows:
```

Comment:
Maybe a small explanation here? Something in the lines of what this strategy can be used for

Reply:
<img width="851" alt="Screenshot 2024-01-30 at 2 48 20 PM" src="https://github.com/Pinelab-studio/pinelab-vendure-plugins/assets/39517388/e7b081ac-9950-472c-ad4b-f5d2f2f76b3b">
 
I believe this will do it